app/views/idv/gpo/index.html.erb

```diff
@@ -9,7 +9,9 @@
       ) %>
 <% end %>
 
-<%= render PageHeadingComponent.new.with_content(@presenter.title) %>
+<%= render AlertComponent.new(message: @presenter.info_alert_message, class: 'margin-bottom-4') %>
```

Comment:
I'm not well-versed in common conventions for presenter classes, but in the case of a simple string (i.e. no conditional logic), I feel like it'd be less indirection to have the text inlined here, rather than adding the new method.

Reply:
Totally agree